### ISSUE TITLE
Chat users not announced by screen reader.

### ISSUE THREAD

**Issue by rkingett** [snikket-im/snikket-ios#203]
When scrolling through an open chat with VoiceOver enabled on iOS, on any version of iOS and the latest version of the app, VoiceOver does not announce who sent the message.

If chat username, and then the message, could all be contained in one VoiceOver line, where it reads the user, then the message, all with one swipe left or right, that would enhance usability dramatically for screen reader users.

## Expected behavior.

With VoiceOver enabled, usernames or display names of message sender should be announced when scrolling through open chat messages.
